src/main/java/core/basesyntax/model/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+import core.basesyntax.ColorSupplier;
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(101),
```

Comment:
magic number, need constant for that
```suggestion
        return new Ball(new Random().nextInt(MAX_NUMBER),
```